twitter/tweets_test.go

```diff
@@ -0,0 +1,71 @@
+package twitter
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestTweetSerialization(t *testing.T) {
+	tweet := loadTestTweet()
+	if tweet.User.ID != 971761 {
+		t.Fail()
+	}
+
+}
+
+func TestLocationSerialization(t *testing.T) {
+	tweet := loadTestLocationTweet()
+	if tweet.User.ID != 971761 {
+		t.Fail()
+	}
+	place := tweet.Place
+	if place == nil {
+		t.Fail()
+	}
+
+	if len(place.BoundingBox.Coordinates) == 0 {
+		t.Fail()
+	}
+	if place.BoundingBox.Type != "Polygon" {
+		t.Fail()
+	}
+	if place.Country != "United States" {
+		t.Fail()
+	}
+	if place.CountryCode != "US" {
+		t.Fail()
+	}
+}
+
+func loadTestTweet() *Tweet {
+	f, e := ioutil.ReadFile("./test_data/tweet.json")
```

Comment:
Would you consider writing `ioutil.ReadFile(filepath.Join("test_data", "tweet.json"))` here so that this Go code is more cross-platform and better for others to read/learn from?